setup_dev_from_wheel.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
+
+# Python environment name
+PYTHON_ENV_NAME=$1
+
+# Exit on any error
+set -e
+
+# Turn off detached head advice on Git
+git config --local advice.detachedHead false
+
+export CATALYST_DIR=$PWD
+export CATALYST_FRONTEND_SRC=$CATALYST_DIR/frontend/catalyst
+export CATALYST_PYTHON_ENV_DIR=/tmp/$PYTHON_ENV_NAME
+
+install_CPL_wheels(){
+    echo "Installing CPL Wheels..."
+
+    # Create (if not created yet) and activate the virtual environment 
+    python3 -m venv $CATALYST_PYTHON_ENV_DIR
+    source $CATALYST_PYTHON_ENV_DIR/bin/activate
+    echo "Success: The Python virtual environment located at '$CATALYST_PYTHON_ENV_DIR' was activated"
+
+    # Clean the Catalyst installation
+    python -m pip uninstall -y pennylane-catalyst
+
+    # Install the CPL suite
+    python -m pip install --extra-index-url https://test.pypi.org/simple/ \
+    pennylane pennylane-lightning pennylane-catalyst --pre --upgrade
+}
+
+checkout_nightly_build(){
+    echo "Checking out nightly build..."
+
+    # Get the Python virtual environment site-packages path
+    export SITEPKGS=$(python3 -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])')
+    export CATALYST_WHEEL=$SITEPKGS/catalyst
+
+    # Update to the latest commit at the main branch
+    cd $CATALYST_DIR
+    git switch main
+    git fetch
+    git pull
+
+    # Search for the commit corresponding to latest available Wheel at TestPyPI
+    git log --grep="bump nightly version" | grep "commit" | cut -d " " -f 2 | while read -r NIGHTLY_BUMP; do
+        # The commit right before the nightly bump must have the same version as the Wheel
+        git checkout $NIGHTLY_BUMP^1; 
+        export DIFF=$(diff $CATALYST_WHEEL/_version.py $CATALYST_FRONTEND_SRC/_version.py)
+        if [ -z "${DIFF}" ]; then
+            export CATALYST_WHEEL_COMMIT=$(git log -1 --format="%h")
+            echo "Success: The commit $CATALYST_WHEEL_COMMIT corresponding to the Wheel was found"
+            break
+        fi
+        echo "Discarding commit, still searching for the corresponding commit to the Wheel..."
+    done
+
+    if [ ! -z "${DIFF}" ]; then
+        echo "Error: Could not find the corresponding commit for the Wheel"
+        exit
+    fi
+}
+
+link_repo_to_wheel(){
+    echo "Linking Catalyst repository to Catalyst Wheel..."
+
+    export SITEPKGS=$(python3 -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])')
+    export CATALYST_WHEEL=$SITEPKGS/catalyst
+
+    # Create hard links to the Wheel Python sources
+    cp -lrf $CATALYST_WHEEL $CATALYST_DIR/frontend/
+}
+
+restore_catalyst_config(){
+    # After linking the Wheel sources, _configuration.py will contain the entry: 'INSTALLED=True'.
+    # Hence, we restore the file from the repository.
+    cd $CATALYST_DIR
+    git checkout frontend/catalyst/_configuration.py
```

Comment:
I was wondering how this would work, but I guess Catalyst still considers `INSTALLED` to be True when importing the package, even though the in the repo it shows up as False 👍

Reply:
I guess because we are not executing the script? Only changing it?